README.md

```
@@ -98,6 +99,22 @@ Postgresql
 
 See online documentation: http://docs.pgrouting.org/latest/en/index.html
 
+## CITATION
+
+To cite pgRouting in publications use:
+
+**BibTeX entry**:
+
+```bibtex
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)